src/audio/multiband_drc/multiband_drc.h

```diff
@@ -38,6 +39,14 @@ struct multiband_drc_comp_data {
 	struct multiband_drc_state state;        /**< compressor state */
 	struct comp_data_blob_handler *model_handler;
 	struct sof_multiband_drc_config *config; /**< pointer to setup blob */
+	/**
+	 * Pointer to the coefficients of the filters used in multiband DRC processing
+	 * which includes crossover, emphasis, and deemphasis filters. These coefficients
+	 * are used by the processing functions and are allocated during the initialization
+	 * phase. This allows efficient access to filter parameters required for audio
+	 * processing across all channels and bands.
+	 */
+	struct multiband_drc_coefficients *coefficients_block; /**< Filter coefficients */
```

Comment:
this *adds* a member of a structure called `coefficients_block` and I don't see it being used anywhere. And this is a 1-commit PR. This doesn't seem right.